examples/bpk-component-nudger/examples.js

```diff
@@ -71,7 +69,7 @@ class NudgerContainer extends Component<
         <BpkNudger
           aria-labelledby="passenger-label"
           id={id}
-          min={1}
+          min={-10}
```

Comment:
I wonder if we should be allowing values to have a minimum less than zero 🤔 can't see the usecase for it

Reply:
Possibly.. but it's up to the consumers what values they want to set.  this is just an example anyway.

Reply:
Can remove this and set it back to 1